dorm/dorm/Presenter/Sidebar/SidebarViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  SidebarbarViewController.swift
+//  dorm
+//
+//  Created by JongHo Park on 2022/07/20.
+//
+
+import UIKit
+
+protocol SidebarDelegate: AnyObject {
+    func showDetailViewController(_ viewController: UIViewController)
+}
+
+final class SidebarViewController: UIViewController {
+
+    // MARK: - Properties
+    private lazy var sidebar: Sidebar = Sidebar()
+    weak var delegate: SidebarDelegate?
+    private var isInitialzed: Bool = false
+
+    // MARK: - ViewController Life cycle
+    init(_ delegate: SidebarDelegate? = nil) {
+        self.delegate = delegate
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Don`t need to be implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        sidebar.tableView.delegate = self
+        sidebar.tableView.dataSource = self
+        navigationController?.navigationBar.prefersLargeTitles = true
+        navigationItem.largeTitleDisplayMode = .always
+        navigationItem.title = "Room manage"
+    }
+
+    override func loadView() {
+        view = sidebar
+    }
+
+}
+
+// MARK: - UITableViewDelegate
+extension SidebarViewController: UITableViewDelegate {
+
+    // cell 을 클릭했을 때 실행되는 delegate method
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        delegate?.showDetailViewController(ScreenState.allCases[indexPath.row].viewController)
+    }
+
+    // cell 을 처음으로 표시할 때, 첫 번째 cell을 selection 상태로 설정
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        guard !isInitialzed else { return }
+
+        if indexPath.row == 0 {
```

Comment:
```suggestion
        guard !isInitialzed, indexPath.row != 0 else { return }
```

아래의 코드를 인덴트 없이 바로 구현할 수 있을것 같아욥

Reply:
77c4dc61d6f603aa9de00456b2274cdf43f99689 에서 수정했습니다!